modules/solid_mechanics/src/vectorpostprocessors/AverageSectionValueSampler.C

```diff
@@ -40,12 +40,19 @@ AverageSectionValueSampler::validParams()
   params.addDeprecatedParam<std::vector<Real>>(
       "lengths",
       {},
-      "Positions along axis of from reference_point at which to compute average values.",
+      "Positions along axis from reference_point at which to compute average values.",
       "Use the 'positions' parameter instead");
   params.addParam<std::vector<Real>>(
       "positions",
       {},
-      "Positions along axis of from reference_point at which to compute average values.");
+      "Positions along axis from reference_point at which to compute average values.");
+  params.addParam<RealVectorValue>(
+      "symmetry_plane",
+      RealVectorValue(0, 0, 0),
+      "Vector normal to a symmetry plane, used if the section has a symmetry plane through it. "
+      "Causes the variables to be treated as components of a vector and zeros out the compomnent "
+      "in the direction normal to symmetry_plane. Three variables must be defined in 'variables' "
+      "to use this option.");
```

Comment:
```suggestion
      "in the direction normal to symmetry_plane. At least four variables must be defined in 'variables' "
      "to use this option, since the three first ones will be used to compute coordinates.");
```

Reply:
No, this isn't how this is intended to be used. The variables wouldn't be the coordinates. The three variables would typically be three components of the displacement, but could be three components of some other vector.

Reply:
so limited to 3 variables in that case?

Reply:
Yes